manager/src/test/java/com/redhat/service/smartevents/manager/api/user/validators/processors/TransformationTemplateConstraintValidatorTest.java

```diff
@@ -0,0 +1,61 @@
+package com.redhat.service.smartevents.manager.api.user.validators.processors;
+
+import javax.inject.Inject;
+
+import org.hibernate.validator.constraintvalidation.HibernateConstraintValidatorContext;
+import org.hibernate.validator.constraintvalidation.HibernateConstraintViolationBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@QuarkusTest
```

Comment:
Likewise; `@QuarkusTest` involves a lot of "heavy lifting" to avoid manual construction of `TransformationTemplateConstraintValidator` (this is why I tend to prefer constructor injection vs field injection as it becomes much easier to construct an instance manually in unit tests.. but that's another discussion completely).

Reply:
Thanks for the trick @manstis . I changed it and it's definitely faster (3 times faster on my machine).